core/shared/src/main/scala/scalaz/zio/Task.scala

```diff
@@ -0,0 +1,341 @@
+package scalaz.zio
+
+import scalaz.zio.Exit.Cause
+import scalaz.zio.internal.{ Executor, Platform }
+
+import scala.concurrent.ExecutionContext
+
+object Task {
+
+  /** See [[scalaz.zio.ZIO.interrupt]]
```

Comment:
All the Scaladoc in all the files looks like it's missing the initial newline after `/**`, I think a quick change to config should fix it! 👍